packages/app-store/zohocrm/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "ZohoCRM",
+  "slug": "zohocrm",
+  "type": "zohocrm_other_calendar",
```

Comment:
@leog Like Hubspot do we need to make it `other_calendar`. I am not sure if I clearly remember when we need to choose other_calendar.